### PR TITLE
Add translatable strings to barrier type field

### PR DESCRIPTION
**Pull request by kjonosm**
for values with usage > 10,000

### CODE REVIEW

File: data/fields/barrier.json [hunk 1]
```diff
@@ -1,5 +1,33 @@
...
+    "label": "Type",
+     "strings": {
+         "options": {
+             "fence": "Fence",
+             "gate": "Gate",
+             "wall": "Wall",
+             "hedge": "Hedge",
+             "kerb": "Kerb",
+             "bollard": "Bollard",
+             "lift_gate": "Lift Gate",
```
Comment (matkoniecz) [inline data/fields/barrier.json:13]: Note that "lift gate", term used in OSM tagging is not used in actual English. 

It seems to be referred to as a boom gate / boom barrier.

https://en.wikipedia.org/wiki/Boom_barrier

File: data/fields/barrier.json [hunk 2]
```diff
@@ -1,5 +1,33 @@
...
+             "swing_gate": "Swing Gate",
+             "cattle_grid": "Cattle Grid",
+             "ditch": "Ditch",
+             "toll_booth": "Toll Booth",
+             "city_wall": "City Wall",
+             "kissing_gate": "Kissing Gate",
+             "chain": "Chain",
+             "jersey_barrier": "Jersey Barrier",
+             "height_restrictor": "Height Restrictor",
+             "sliding_gate": "Sliding Gate",
```
Comment (matkoniecz) [inline data/fields/barrier.json:29]: Note that barrier=sliding_gate is used regionally by paid mappers. I would not include it as a supported value.

**Review by matkoniecz — CHANGES_REQUESTED** [review]

File: data/fields/barrier.json (hunk 1 above)
Comment (kjonosm) [reply data/fields/barrier.json]: Fixed

File: data/fields/barrier.json (hunk 2 above)
Comment (kjonosm) [reply data/fields/barrier.json:29]: I will leave it out for now, though I don't really have a problem with it as the value is well documented and in widespread use...

Comment (matkoniecz) [reply data/fields/barrier.json]: There is a high usage count but usage is not actually widespread.

See also https://taginfo.openstreetmap.org/tags/barrier=sliding_gate#chronology

screen01

Comment (kjonosm) [reply data/fields/barrier.json]: Distribution doesn't look too bad though, see https://taginfo.openstreetmap.org/tags/barrier=sliding_gate#map

File: data/fields/barrier.json [hunk 3]
```diff
@@ -1,5 +1,32 @@
...
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+     "strings": {
+         "options": {
+             "fence": "Fence",
+             "gate": "Gate",
+             "wall": "Wall",
+             "hedge": "Hedge",
+             "kerb": "Kerb",
```
Comment (tyrasd) [inline data/fields/barrier.json:11]:
```suggestion
             "kerb": "Curb",
```

_Kerb_ is British English, if I'm not mistaken.

File: data/fields/barrier.json [hunk 4]
```diff
@@ -1,5 +1,32 @@
...
+             "bollard": "Bollard",
+             "lift_gate": "Boom Barrier",
```
Comment (tyrasd) [inline data/fields/barrier.json:13]:
```suggestion
             "lift_gate": "Boom Gate",
```

to keep it similar to other types of gates.

**Review by tyrasd — CHANGES_REQUESTED** [review]
see below

File: data/fields/barrier.json [hunk 5]
```diff
@@ -1,5 +1,32 @@
 {
     "key": "barrier",
     "type": "typeCombo",
-    "label": "Type"
+    "label": "Type",
+     "strings": {
```
Comment (tyrasd) [inline data/fields/barrier.json:5]: PS: the indentation does have one too many space character here and the lines below.

```suggestion
    "strings": {
```

**Comment by kjonosm** [comment]
@tyrasd thanks for the review. I updated the pr accordingly.

**Pull request merged**